### ISSUE TITLE
Accessibility issue: <html> element must have a lang attribute on /az_core/mobile_nav_callback/menu_link_content%3A911b448b-1e04-4a83-bc38-9a13fa16224b

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#70]
## What
An accessibility scan flagged the element `<html>` because <html> element must have a lang attribute. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/html-has-lang?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/d014f540-6137-4e66-ab8b-d364279e1fcc.png)

To fix this, ensure every html document has a lang attribute.

Specifically:

Fix any of the following:
-   The <html> element does not have a lang attribute

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
